README.md

```diff
@@ -18,7 +18,7 @@ reason, just add the following line to your `deps` in the `mix.exs` and run
 `mix deps.get`.
 
 ```elixir
-{:membrane_raw_video_format, "~> 0.4.1"}
+{:membrane_raw_video_format, "~> 0.5.0"}
```

Comment:
```suggestion
{:membrane_raw_video_format, "~> 0.4.2"}
```

Reply:
I know that technically speaking we should bump minor, but I highly doubt that somebody is relying on the fact that `frame_size` returns `{:error, :invalid_dimensions}` for 10 bits formats.
I think just bumping the patch is fine and this way we will also avoid problems with incompatible dependencies among the plugins that use `membrane_raw_video_format`